modules/apps/commerce/commerce-shipment-web/src/main/java/com/liferay/commerce/shipment/web/internal/portlet/action/EditCommerceShipmentItemMVCActionCommand.java

```diff
@@ -247,6 +247,9 @@ private CommerceShipmentItem _updateCommerceShipmentItem(
 			long commerceInventoryWarehouseId =
 				commerceInventoryWarehouse.getCommerceInventoryWarehouseId();
 
+			String inventoryWarehouseExternalReferenceCode =
```

Comment:
I think this should be commerceInventoryWarehouseExternalReferenceCode and should go before the *Id (without any empty line between them)